From 2815aa97c3b0fdf15ca6845c561a03e8ca9ddb09 Mon Sep 17 00:00:00 2001
From: Dimitar Maznekov <d.maznekov@optixco.com>
Date: Fri, 29 Oct 2021 14:56:11 +0300
Subject: [PATCH 2/3] Remove compile options LIBSOUP_LIBS in gstd

---
 gstd/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/gstd/Makefile.am b/gstd/Makefile.am
index 5a90e28..5fbee22 100644
--- a/gstd/Makefile.am
+++ b/gstd/Makefile.am
@@ -84,8 +84,7 @@ gstd_LDFLAGS = $(GST_LIBS)					\
 	       $(GIO_UNIX_LIBS)					\
 	       $(GJSON_LIBS)					\
 	       $(LIBD_LIBS)					\
-	       $(LIBSOUP_LIBS)					\
-	       -Wl,-rpath -Wl,$(libdir)
+	       $(LIBSOUP_LIBS)
 
 gstd_LDADD = libgstd-core.la
 
-- 
2.30.2

